### PR TITLE
Register new snapshots

### PR DESCRIPTION
**Pull request by alexcrichton**

### CODE REVIEW

File: src/libcore/intrinsics.rs [hunk 1]
```diff
@@ -198,12 +178,8 @@ extern "rust-intrinsic" {
     pub fn min_align_of<T>() -> usize;
     pub fn pref_align_of<T>() -> usize;
 
-    /// Get a static pointer to a type descriptor.
-    #[cfg(stage0)] // SNAP 270a677
-    pub fn get_tydesc<T: ?Sized>() -> *const TyDesc;
-
     /// Gets a static string slice containing the name of a type.
-    #[cfg(not(stage0))] // SNAP 270a677
+    #[cfg(not(stage0))]
```
Comment (eddyb) [inline src/libcore/intrinsics.rs]: Why wasn't the attribute removed entirely?

Comment (alexcrichton) [inline src/libcore/intrinsics.rs]: I forget :(

**Pull request merged**